tests/lib/boot.sh

```diff
@@ -33,3 +33,15 @@ bootenv_unset() {
         fw_setenv "$var"
     fi
 }
+
+get_boot_path() {
+    if [ -d /boot/uboot/ ] && ! [ -z "$(ls -A /boot/uboot/)" ]; then
+        echo "/boot/uboot/"
```

Comment:
Why echo'ing and not just returning the value?

Reply:
Shell functions can only return integers. The rest must go through `echo`